Hardware/CPU/IntelCPU.cs

```diff
@@ -289,6 +296,7 @@ public IntelCPU(int processorIndex, CPUID[][] cpuid, ISettings settings)
         if (Ring0.Rdmsr(MSR_RAPL_POWER_UNIT, out eax, out edx))
           switch (microarchitecture) {
             case Microarchitecture.Silvermont:
+            case Microarchitecture.Airmont:

```

Comment:
Just read this out of context, but it seems you have added a new case and not reproduced the code inside the case?
I believe this will make SilverMont stop working, as it's now an "empty case" ?